lib/src/main/java/at/bitfire/cert4android/CustomCertStore.kt

```diff
@@ -77,7 +77,13 @@ class CustomCertStore internal constructor(
     /**
      * Determines whether a certificate chain is trusted.
      */
-    fun isTrusted(chain: Array<X509Certificate>, authType: String, trustSystemCerts: Boolean, appInForeground: StateFlow<Boolean>?): Boolean {
+    fun isTrusted(
```

Comment:
We should document how timeouts and scope are handled. And add KDoc for the parameters